src/docbuilder/rustwide_builder.rs

```diff
@@ -230,19 +211,29 @@ impl RustwideBuilder {
                     .prefix("essential-files")
                     .tempdir()?;
 
-                let files = ESSENTIAL_FILES_VERSIONED
+                let toolchain_version = self.toolchain.as_dist().unwrap().name();
```

Comment:
```suggestion
                let toolchain_version = self.toolchain.as_dist().expect("we only use the Toolchain::dist constructor").name();
```

maybe worth it 🤷. It'd be nice to have it statically known that it is a dist toolchain, but I guess that'd require rustwide changes.